common/lib/dependabot/file_fetchers/base.rb

```diff
@@ -890,10 +890,10 @@ def find_submodules(path)
       sig { params(path: String).returns(T::Boolean) }
       def isLfsEnabled(path)
         filepath = File.join(path,".gitattributes")
-        lfsEnabled = File.exist?(filepath) && File.readable?(filepath) && SharedHelpers.run_shell_command("cat #{filepath} | grep \"filter=lfs\"").include? "#{filepath}"
+        lfsEnabled = T.let(true, T::Boolean) if File.exist?(filepath) && File.readable?(filepath) && SharedHelpers.run_shell_command("cat #{filepath} | grep \"filter=lfs\"").include? "filter=lfs"
```

Comment:
any reason for copying the same code pattern in 3 different files? Can we centralize it in one location?